### ISSUE TITLE
Remove Creator Showcase from gatsbyjs.org

### ISSUE THREAD

**Issue by tesseralis** [gatsbyjs/gatsby#19362]
## Summary

Remove https://gatsbyjs.org/creators and all links to it.

### Motivation

* Its functionality has been partially overtaken by https://gatsbyjs.org/showcase
* It's less code complexity, less images, and will help the site build faster (See: #19105)
* The current list of creators skews *heavily* toward white men (and companies), and isn't representative of the Gatsby community

**Comment by fk** [comment]
(Thinking out loud after taking a quick look)

There's currently 38 entires in the `creators.yml` file.

AFAICS the only place we link to "Creators" from is https://www.gatsbyjs.org/contributing/submit-to-creator-showcase/. As I remember things, those contributing guidelines might be the one place where we ever talked about this section of .org? Not sure though …

While my spontaneous vote would have been "yes, let's remove it — this feels like dead code for a (long) while now", the idea (still) makes sense to me. And a couple of things that would increase exposure/value seem quite low-hanging:

- We do map showcase site authors to creators already (and link to showcase items from the creator's profile, see e.g. https://www.gatsbyjs.org/creators/people/lekoarts), but we don't map starters yet.
- We don't link from showcase entries to the creator profile though.
- We already have bespoke guidelines to add to "Creators", and mention the mapping to "Showcase" entries there, but we don't do so vice versa in the "Showcase" contributing guidelines.

@marcysutton @shannonbux @calcsam @KyleAMathews How do you feel about this?

**Comment by muescha** [comment]
To spin more the idea from @fk:

- Remove the creators index -> redirect to showcase
- Put the current content of creators detail page 1:1 under each schowcase where it is the creator. - If there is more than 1 showcase then list also the other showcases as preview and link. 

- (As temporary?) we add a "moved permanent" Redirect from the contributors detail to the first showcase

**Comment by shannonbux** [comment]
Yeah, I think just deleting it and redirecting to site showcase is a great idea; I wouldn't worry about repurposing the content as long as we notify the 38 people that we're working on a better version of this in the future? 

Since it's not linked to from anywhere else, I doubt many people visit it if at all!

**Comment by Yofamb** [comment]
I'm happy to work on this issue once everyone has decided the best approach

**Comment by Yofamb** [comment]
I've created a draft pull request with a first pass on removing the creator showcase. All tests pass and I believe i've removed everything. More thought needs go on the redirects.

**Comment by sidharthachatterjee** [comment]
We've decided to remove the pages and links to it but keep the data for now.

**Comment by EnKrypt** [comment]
As @shannonbux suggested, I think some sort of notification would have been nice.

I was wondering why the creator showcase was showing a 404 before finding this issue. As of now, it does not redirect.

**Comment by Yofamb** [comment]
I guess I missed doing the redirect from `/creators` to `/showcase` perhaps i'll do a PR for the fix.

**Comment by muescha** [comment]
i tried to use this splat for an redirect but don't get it work on my local machine with `develop` and `build`/`serve`:

```jsx
  createRedirect({
    fromPath: `/creators/*`,
    toPath: `/showcase`,
    isPermanent: true,
  })
```

i see some configurations about the usage of `/oldpage/*` in the issues, but maybe it works only on `netlify`?

see https://github.com/gatsbyjs/gatsby/issues/3782#issuecomment-505981236

> For anyone interested to redirect with wildcard, you just need to ensure you have the correct host config (netlify/apache. Gatsby does not really filter it out so it gives us some freedom.
> 
> For example:
> 
> ```
> createRedirect({
>   fromPath: "/someLegacyPath/*",
>   toPath: "https://newDomain.com/someLegacyPath/:splat",
> })
> ```
> 
> `:splat` is something specific for Netlify redirects. (see [netlify.com/docs/redirects/#splats](https://www.netlify.com/docs/redirects/#splats))
> Your redirect won't be "portable" but at least it will work in Netlify.

**Comment by coreyward** [comment]
I disagree that the motives cited provide adequate justification for the actions taken here. 

> ### Motivation
> * Its functionality has been partially overtaken by https://gatsbyjs.org/showcase

Isn't the creators showcase *newer* than the site showcase? In any case, I cannot link to any page on the Gatsby Showcase that shows potential clients a list of projects I've built, which I did with the Creators Showcase. Similarly, if I don't have bandwidth for a project, I typically send companies to the Creators Showcase to find other developers. 

> * It's less code complexity, less images, and will help the site build faster (See: #19105)

Consider instead using [incremental builds](https://www.gatsbyjs.com/cloud/).

> * The current list of creators skews _heavily_ toward white men (and companies), and isn't representative of the Gatsby community

[The demographic makeup of web developers skews _heavily_ toward white men.](https://insights.stackoverflow.com/survey/2019#demographics) The Gatsby community can be diverse, accepting, and help shine a light on others without efforts to deliberately hinder the advancement white men that are part of that community.
